osu.Game.Tests/Visual/UserInterface/TestSceneUpdateableBeatmapBackgroundSprite.cs

```diff
@@ -32,7 +32,7 @@ private void load(OsuGameBase osu, IAPIProvider api, RulesetStore rulesets)
             this.api = api;
             this.rulesets = rulesets;
 
-            testBeatmap = ImportBeatmapTest.LoadOszIntoOsu(osu);
+            testBeatmap = ImportBeatmapTest.LoadOszIntoOsu(osu).Result;
```

Comment:
Calling `.Result` is basically a `Wait()` call, so the same applies as my previous comment.

Reply:
The same response applies as above.